consensus/core/src/block_manager.rs

```diff
@@ -105,15 +105,15 @@ impl BlockManager {
         let dag_state = self.dag_state.read();
 
         // If block has been already received and suspended, or already processed and stored, or is a genesis block, then skip it.
-        if self.suspended_blocks.contains_key(&block_ref) || dag_state.contains_block(&block_ref)? {
+        if self.suspended_blocks.contains_key(&block_ref) || dag_state.contains_block(&block_ref) {
             return Ok(None);
```

Comment:
Probably the method doesn't have a reason to fail (return an err) any more, so I guess we could change the methods to not return a `ConsensusResult`